PowerShell/ScubaGear/Sample-Config-Files/omit_policies.yaml

```
@@ -20,7 +20,7 @@ OutReportName: BaselineReports
 OmitPolicy:
   MS.EXO.2.2v1:
     Rationale: "Known false positive; our SPF policy currently cannot to be retrieved via ScubaGear due to a split
-      horizon setup but is available publicly."
+    horizon setup but is available publicly."
```

This change results in invalid YAML, the indent is actually needed.

---

The indent is still missing
```suggestion
      horizon setup but is available publicly."
```